### PR TITLE
Check for pairwise (`overflow-x`, `width`) and (`overflow-y`, `height`) handling in `isPossiblyScrollable()`

### PR DESCRIPTION
**Pull request by kasperisager**
- [ ] Follow up on DEV-10932.

### CODE REVIEW

File: packages/alfa-rules/src/sia-r84/rule.ts [hunk 1]
```diff
@@ -62,36 +62,39 @@ export namespace Outcomes {
...
 function isPossiblyScrollable(device: Device): Predicate<Element> {
-  /**
-   * Determine if an element is possibly scrollable. This is determined by the
-   * following factors:
-   *
-   * - A computed `width` or `height` that is not `auto`.
-   * - A computed `overflow-x` or `overflow-y` that is `auto`, `clip`, or
-   *   `scroll`.
-   */
+  const propertiers = [
```
Comment (Jym77) [inline packages/alfa-rules/src/sia-r84/rule.ts:74]: "propertiers"?
```suggestion
  const properties = [
```

File: packages/alfa-rules/src/sia-r84/rule.ts [hunk 2]
```diff
@@ -62,36 +62,39 @@ export namespace Outcomes {
...
-      const { value: overflow } = style.computed(property);
-
-      switch (overflow.value) {
-        case "auto":
-        case "clip":
-        case "scroll":
-          return true;
-      }
-    }
+    return propertiers.some(
```
Comment (Jym77) [inline packages/alfa-rules/src/sia-r84/rule.ts:82]:
```suggestion
    return properties.some(
```

File: packages/alfa-rules/src/sia-r84/rule.ts [hunk 3]
```diff
@@ -62,36 +62,39 @@ export namespace Outcomes {
   );
 }
 
+/**
+ * Determine if an element is possibly scrollable. This is determined by the
+ * following factors:
+ *
+ * - A computed `width` or `height` that is not `auto`.
+ * - A computed `overflow-x` or `overflow-y` that is `auto`, `clip`, or
+ *   `scroll`.
```
Comment (Jym77) [inline packages/alfa-rules/src/sia-r84/rule.ts:71]: Comment should be updated and made more accurate to reflect the pairing.

**Review by Jym77 — CHANGES_REQUESTED** [review]

**Comment by kasperisager** [comment]
Good catches, thanks!

**Review by Jym77 — APPROVED** [review]

**Pull request merged**